### DISCUSSION TITLE
Homework 3 feedback list

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#14] [Announcements]
Homework 03 [FeedbackList](https://github.com/STAT5125-UConn/2022Spring/blob/main/homework/hw03/FeedbackList.md) is released in 4276dea.

**Comment by AeliaPrima** [comment]
Hello Dr. Wang,

Last night, I sent the invitation link to other two students for hw03 review. However, I do not think either of them are able to see the link as usual as checked with one of them if she received the invitation. A few moment ago, WentingS told me she sent her invitation but I am not able to see it either.

I am not sure if this is happening to all of us or just me and my two reviewers. Do you have any idea?

Regards,
Yiran

**Reply by Ossifragus** [reply]
It seems these two students already have access to your repository. They need to you the link to your repository to submit their review.

**Reply by Xiw2000** [reply]
I think this time everyone will not receive the invitation link. You can see other people's hw3 directly in the organization STAT5125-UConn(if you have access to it).

Xinhao

**Comment by andyslater** [comment]
Hi Sorry, I do not understand what you are asking?

I have provided access to my repo. When then get the permission the link to the repo should be in their list of repos, right?

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

**Reply by Ossifragus** [reply]
When you give other students access to your repository, they should receive an invitation email. In case the emails are not delivered, you can send the link to your repository to them directly.

**Comment by andyslater** [comment]
Hi,

 

I am very sorry but I made updates to my code and MD output. I did not provide an MD file till yesterday, and I have modified it again today.

I’ve tried to indicate in my code where the changes were made. Since the submission date.

 

I believe this is a link to the file submitted on 3/7

https://github.com/STAT5125-UConn/2022-hw03-andyslater/commit/074d24daf805ec8c81c532e03620662fe3da5e66

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002
